### PR TITLE
add another exception for github URLs for "osc add"

### PR DESCRIPTION
**Pull request by adrianschroeter**
osc#905

### CODE REVIEW

**Comment by lethliel** [comment]
This looks good to me
@marcus-h any objections in merging this?

**Comment by marcus-h** [comment]
On 2022-04-14 01:32:57 -0700, Marco Strigl wrote:
> This looks good to me
> @marcus-h any objections in merging this?
>
No:)

File: osc/commandline.py [hunk 1]
```diff
@@ -4819,7 +4819,9 @@ def do_add(self, subcmd, opts, *args):
 
         # Do some magic here, when adding a url. We want that the server to download the tar ball and to verify it
         for arg in parseargs(args):
-            if arg.endswith('.git') or arg.startswith('git://') or arg.startswith('git@') or arg.startswith('https://github.com') or arg.startswith('https://gitlab.com'):
+            if arg.endswith('.git') or arg.startswith('git://') or \
+               arg.startswith('git@') or (arg.startswith('https://github.com') and not '/releases/' in arg) or \
```
Comment (dirkmueller) [inline osc/commandline.py:4823]:
```suggestion
               arg.startswith('git@') or (arg.startswith('https://github.com') and '/releases/' not in arg) or \
```

**Comment by dmach** [comment]
Looks good to me, merging. We'll eventually move the `not` afterwards.

**Pull request merged**